docs/abci-cli.rst

```diff
@@ -1,5 +1,5 @@
-Using the abci-cli
-==================
+Using ABCI
```

Comment:
This really should be the ABCI-CLI ... using ABCI is something an app dev does. Here we're talking about using the CLI